### PR TITLE
Misuv 8373 single year confirmation page

### PR DESCRIPTION
**Pull request by tharunprasath**

### CODE REVIEW

File: app/controllers/optIn/ConfirmTaxYearController.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import play.api.mvc._
+import services.DateService
+import services.optIn.OptInService
+import utils.AuthenticatorPredicate
+import views.html.optIn.ConfirmTaxYear
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ConfirmTaxYearController @Inject()(val view: ConfirmTaxYear,
```
Comment (james-king-hmrc) [inline app/controllers/optIn/ConfirmTaxYearController.scala:38]: there is already OptInCompletedController why not update that class with single-year work?

File: app/views/optIn/ConfirmTaxYear.scala.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
```
Comment (james-king-hmrc) [inline app/views/optIn/ConfirmTaxYear.scala.html:2]: this file could be named 'OptInCompletedSingleYearView' inline with existing file name

File: app/controllers/optIn/ConfirmTaxYearController.scala (hunk 1 above)
Comment (james-king-hmrc) [reply app/controllers/optIn/ConfirmTaxYearController.scala:38]: or may be renamed it 'OptInCompletedSingleYearController'

Comment (james-king-hmrc) [reply app/controllers/optIn/ConfirmTaxYearController.scala:38]: Okay, sounds like I misunderstood

**Review by JordHemsworth — APPROVED** [review]
looks good!

**Pull request merged**